forage-android/src/main/java/com/joinforage/forage/android/network/ForageConstants.kt

```diff
@@ -36,4 +37,10 @@ internal object ForageConstants {
     object VGS {
         const val PIN_FIELD_NAME = "pin"
     }
+
+    object ErrorResponseObjects {
+        val INVALID_PIN_ERROR = listOf(
```

Comment:
folks can mistake this for INVALID_PIN, like `ebt_error_55` which could lead to misuse, can we use `INCOMPLETE_PIN_ERROR`?

Reply:
Why yes we can! :)